asv/plugins/mercurial.py

```diff
@@ -169,11 +169,10 @@ def get_date_from_name(self, name):
 
     def get_branch_commits(self, branch):
         if self._repo.version >= (4, 5):
-            query = "branch({0})"
+            query = f"branch({self.get_branch_name(branch)})"
```

Comment:
The original is ok, you can remove the `0` and just leave `{}`, but don't convert to a f-string